css/style.css

```diff
@@ -723,6 +740,28 @@ body:not(.loaded) > *:not(#loader-wrapper) {
 					background: rgba(58,57,57,0.025);
 				}
 
+        input[type="button"].style4,
+  		  input[type="submit"].style4,
+  		  input[type="reset"].style4,
+        .button.style4
+        {
+          padding: 2px 8px;
+          text-align: center;
+          color: white;
+          background-color: #212121;

```

Comment:
so much inconsitencies between tabs and spaces, do a find all and replace the spaces with tabs. N++ can do it if you use the regex mode, replacing with `/t`